raiden/storage/serialization/cache.py

```diff
@@ -19,25 +19,31 @@ def class_type(instance: Any) -> str:
     return f"{instance.__class__.__module__}.{instance.__class__.__name__}"
 
 
-def set_class_type(schema, data, instance, **kwargs):  # pylint: disable=unused-argument
+def set_class_type(  # pylint: disable=unused-argument
+    schema: Schema, data: dict, instance: Any, **kwargs: Any
+) -> dict:  # pylint: disable=unused-argument
```

Comment:
Do we need this pylint annotation twice?

Reply:
No, this is just a result of black moving the comment to a location where it does not apply and me not noticing it. Will fix.